paddle/fluid/inference/tests/api/config_printer.h

```diff
@@ -62,7 +62,7 @@ std::ostream &operator<<(std::ostream &os,
                          const contrib::AnalysisConfig &config) {
   os << GenSpaces(num_spaces) << "contrib::AnalysisConfig {\n";
   num_spaces++;
-  os << *reinterpret_cast<const NativeConfig *>(&config);
+  os << config.ToNativeConfig();
   if (!config.model_from_memory()) {
     os << GenSpaces(num_spaces) << "prog_file: " << config.prog_file() << "\n";
     os << GenSpaces(num_spaces) << "param_file: " << config.params_file()
```

Comment:
`prog_file` and `param_file` are printed in `NativeConfig`, so line 67 and 68 can be removed.

Reply:
`NatvieConfig` don't print `prog_file` and `param_file`, it only print `model_dir`
```
I0108 19:15:03.677664   894 tester_helper.h:63] NativeConfig {
  PaddlePredictor::Config {
    model_dir:
  }
  use_gpu: 0
  device: 0
  fraction_of_gpu_memory: 0
  specify_input_name: 1
  cpu_num_threads: 1
}
```

Reply:
Sorry. It is because I add the print of `prog_file` and `param_file` in my PR. I will delete them.